auslib/dockerflow.py

```diff
@@ -26,8 +32,14 @@ def heartbeat():
         """Per the Dockerflow spec:
         Respond to /__heartbeat__ with a HTTP 200 or 5xx on error. This should
         depend on services like the database to also ensure they are healthy."""
-        database_entry_value = heartbeat_database_fn(dbo)
-        return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        try:
+            database_entry_value = heartbeat_database_fn(dbo)
+            return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        except Exception as e:
+            logging.exception("Error Occured %s", e.message)
+            return Response(status=502, response="Error occured")
+            if sentry.client:
+                sentry.captureMessage(e.message)
```

Comment:
Same here - Sentry will automatically capture exceptions if a 500 is returned. This is after the return statement anyways, so it will never run :)